projects/1.18/assets/furnish/furnish/lang/zh_cn.json

```
@@ -1 +1,322 @@
+  "block.furnish.blue_awning": "蓝色棚",
+  "block.furnish.brown_awning": "棕色棚",
+  "block.furnish.cyan_awning": "青色棚",
+  "block.furnish.gray_awning": "灰色棚",
+  "block.furnish.green_awning": "绿色棚",
+  "block.furnish.light_blue_awning": "淡蓝色棚",
+  "block.furnish.light_gray_awning": "淡灰色棚",
+  "block.furnish.lime_awning": "黄绿色棚",
+  "block.furnish.magenta_awning": "品红色棚",
+  "block.furnish.orange_awning": "橙色棚",
+  "block.furnish.pink_awning": "粉红色棚",
+  "block.furnish.purple_awning": "紫色棚",
+  "block.furnish.red_awning": "红色棚",
+  "block.furnish.yellow_awning": "黄色棚",
+
+  "block.furnish.amphora": "双耳瓶",
+  "block.furnish.white_amphora": "白色双耳瓶",
+  "block.furnish.black_amphora": "黑色双耳瓶",
+  "block.furnish.blue_amphora": "蓝色双耳瓶",
+  "block.furnish.brown_amphora": "棕色双耳瓶",
+  "block.furnish.cyan_amphora": "青色双耳瓶",
+  "block.furnish.gray_amphora": "灰色双耳瓶",
+  "block.furnish.green_amphora": "绿色双耳瓶",
+  "block.furnish.light_blue_amphora": "淡蓝色双耳瓶",
+  "block.furnish.light_gray_amphora": "淡灰色双耳瓶",
+  "block.furnish.lime_amphora": "黄绿色双耳瓶",
+  "block.furnish.magenta_amphora": "品红色双耳瓶",
+  "block.furnish.orange_amphora": "橙色双耳瓶",
+  "block.furnish.pink_amphora": "粉红色双耳瓶",
+  "block.furnish.purple_amphora": "紫色双耳瓶",
+  "block.furnish.red_amphora": "红色双耳瓶",
+  "block.furnish.yellow_amphora": "黄色双耳瓶",
+
+  "block.furnish.book_pile": "书堆",
+
+  "block.furnish.red_bunting": "红橙彩旗",
+  "block.furnish.yellow_bunting": "蓝黄彩旗",
+  "block.furnish.green_bunting": "紫黄绿彩旗",
+  "block.furnish.lantern_bunting": "灯笼彩旗",
+  "block.furnish.soul_lantern_bunting": "灵魂灯笼彩旗",
+
+  "block.furnish.white_paper_lamp": "白色纸灯笼",
+  "block.furnish.black_paper_lamp": "黑色纸灯笼",
+  "block.furnish.blue_paper_lamp": "蓝色纸灯笼",
+  "block.furnish.brown_paper_lamp": "棕色纸灯笼",
+  "block.furnish.cyan_paper_lamp": "青色纸灯笼",
+  "block.furnish.gray_paper_lamp": "灰色纸灯笼",
+  "block.furnish.green_paper_lamp": "绿色纸灯笼",
+  "block.furnish.light_blue_paper_lamp": "淡蓝色纸灯笼",
+  "block.furnish.light_gray_paper_lamp": "淡灰色纸灯笼",
+  "block.furnish.lime_paper_lamp": "黄绿色纸灯笼",
+  "block.furnish.magenta_paper_lamp": "品红色纸灯笼",
+  "block.furnish.orange_paper_lamp": "橙色纸灯笼",
+  "block.furnish.pink_paper_lamp": "粉红色纸灯笼",
+  "block.furnish.purple_paper_lamp": "紫色纸灯笼",
+  "block.furnish.red_paper_lamp": "红色纸灯笼",
+  "block.furnish.yellow_paper_lamp": "黄色纸灯笼",
+
+  "block.furnish.metal_mailbox": "金属邮箱",
+  "msg.furnish.mailbox.ownership_info": "再次右击以声明所有权",
+  "msg.furnish.mailbox.set_owner": "这邮箱是你的",
+  "msg.furnish.mailbox.no_permission": "这邮箱不是你的",
+  "msg.furnish.mailbox.mail_delivered": "信件已送达",
+  "msg.furnish.mailbox.mail_delivered_to": "信件已送达至 %s",
+  "msg.furnish.mailbox.full": "邮箱满了",
+  "msg.furnish.mailbox.new_mail": "收到信件",
+  "msg.furnish.mailbox.new_mail_loc": "收到信件在 %s",
+  "msg.furnish.mailbox.invalid_mail": "无法寄这个",
+  "subtitle.furnish.mailbox_update": "邮箱切换",
+  "item.furnish.letter": "信件",
+  "msg.furnish.letter.attachment_added": "附加 %s 到信",
+  "msg.furnish.letter.attachment_removed": "删除附件",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "msg.furnish.letter.attachment_added": "已把 %s 附到信中",
  "msg.furnish.letter.attachment_removed": "附件已删除",
```
请确认这个massage是不是在这两个操作之后显示的